client/src/components/Auth/Login.tsx

```diff
@@ -1,8 +1,8 @@
-import { useEffect } from 'react';
+import { useState, useEffect } from 'react';
 import LoginForm from './LoginForm';
 import { useAuthContext } from '~/hooks/AuthContext';
 import { useNavigate } from 'react-router-dom';
-import { SHOW_GOOGLE_LOGIN_OPTION, ALLOW_REGISTRATION, DOMAIN_SERVER } from "~/utils/envConstants";
+import { SHOW_GOOGLE_LOGIN_OPTION, ALLOW_OPENID, VITE_OPENID_LABEL, VITE_OPENID_URL, ALLOW_REGISTRATION, DOMAIN_SERVER } from "~/utils/envConstants";
```

Comment:
I do not mind updating the code. What is the best way to resync my fork with the current main when there are conflicts?  Last time I did another branch and squashed my changes and resubmitted it.  However, that is probably not the proper method.

Reply:
It should be pretty easy to update once you've synced it.
Steps that I follow:
- I usually just click the sync fork on the main branch from gjthub when I see that its behind
- Then `git checkout main`
- `git pull`
- `git checkout <your_branch>`
- and either `git rebase main` or `git merge main`
I prefer rebase as it makes for a much cleaner commit tree and allows me to step through the commits while resolving conflicts one at a time. It also gives a bit more control in case I want to abort. However, some people find this process difficult if you haven't done it before, in which case you might choose to just merge.